### PR TITLE
Homepage

### PR DESCRIPTION
**Pull request by kjasani111**
I finished the full homepage.

### CODE REVIEW

**Comment by kjasani111** [comment]
Simulator Screen Shot - iPhone 11 Pro Max - 2019-11-25 at 21 35 54
Simulator Screen Shot - iPhone 11 Pro Max - 2019-11-25 at 21 35 59
Simulator Screen Shot - iPhone 11 Pro Max - 2019-11-25 at 21 36 06

File: carriage-user-ios/carriage-user-ios/RideHistoryCell.swift [hunk 1]
```diff
@@ -0,0 +1,129 @@
+//
+//  RideHistoryCell.swift
```
Comment (lsizemore8) [inline carriage-user-ios/carriage-user-ios/RideHistoryCell.swift:2]: Take out any copyright comments with individual names

File: carriage-user-ios/carriage-user-ios/SettingsLauncher.swift [hunk 2]
```diff
@@ -0,0 +1,123 @@
+//
+//  SettingsLauncher.swift
```
Comment (lsizemore8) [inline carriage-user-ios/carriage-user-ios/SettingsLauncher.swift:2]: Remove here as well

File: carriage-user-ios/carriage-user-ios/SettingsLauncher.swift [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+//  carriage-user-ios
+//
+//  Created by Kinjal Jasani on 11/25/19.
+//  Copyright © 2019 Beth Mieczkowski. All rights reserved.
+//
+
+import UIKit
```
Comment (lsizemore8) [inline carriage-user-ios/carriage-user-ios/SettingsLauncher.swift:8]: Is there any reason why we're importing UIKit and not SnapKit?

Comment (kjasani111) [reply carriage-user-ios/carriage-user-ios/SettingsLauncher.swift:8]: import UIKit is the default statement to put in when you create a new file. I didn't need SnapKit in this particular file so I didn't import it.

File: carriage-user-ios/carriage-user-ios/RideHistoryCell.swift (hunk 1 above)
Comment (kjasani111) [reply carriage-user-ios/carriage-user-ios/RideHistoryCell.swift:2]: Done

File: carriage-user-ios/carriage-user-ios/SettingsLauncher.swift (hunk 2 above)
Comment (kjasani111) [reply carriage-user-ios/carriage-user-ios/SettingsLauncher.swift:2]: Done

**Pull request closed without merging**